plugins/catalog-backend-module-msgraph/src/processors/MicrosoftGraphOrgEntityProvider.test.ts

```diff
@@ -131,7 +147,7 @@ describe('MicrosoftGraphOrgEntityProvider', () => {
               type: 'team',
             },
           },
-          locationKey: 'msgraph-org-provider:test',
+          locationKey: 'msgraph-org-provider:customProviderId',
```

Comment:
Can we confirm that a user with database persistence(postgres) are able to move from the old format to the new one without experiencing conflicts with entities having a different locationKey.
Slightly worried that the catalog will be unhappy but I'm not sure so it's probably worth testing.

Reply:
The format is not changing. "test" was just used at the test as the value of `target` was used as `id`. Users could still use this as provider ID.

Switching shouldn't be a big deal though I guess. If a provider was removed all its data will be removed and the new provider will readd them. There might be a moment of conflict where old data was not removed yet but the new provider tries to add the new data already as well as potentially broken relationships/references until this was resolved, but eventually this should resolve itself.